Dockerfile

```diff
@@ -25,6 +25,7 @@ FROM wrouesnel/postgres_exporter:latest AS postgres_exporter
 FROM postgres:${PG_VERSION}
 ARG VERSION 
 ARG POSTGIS_MAJOR=3
+ARG WALG_VERSION=1.1
```

Comment:
WALG just released v2. Might be worth starting there?

Reply:
Good idea!